### DISCUSSION TITLE
Jarallax.js - Parallax Image slider - https://github.com/phrizm/bs-jarallax

### DISCUSSION THREAD

**Discussion by ritchiemarkov** [bootscore/bootscore#349] [Q&A] [answered]
Hello Basti and Happy new Year!

First of all I think it is very useful as a possible addition to the theme a possibility to have the images with the parallax effect. For this reason I used the jarallax.js (https://jarallax.nkdev.info/ - https://github.com/nk-o/jarallax/) library which is quite easy to use and customize.

For this puropose i need your help/contribution if you have any time. 
Now i'll describe what i have done/archived and what is left "to-do".

Drop .php file into and folder into child theme

functions.php 

// Jarallax - Parallax effects
require_once dirname(__FILE__) . '/wpt-jarallax.php';

Process:

Add image to the post - add simply add a custom css class (parallax).

And voila, its add's a div with h-100 / w-100 class (wich can be customized into javascript part) with our image.

What is my problem: i dont know why but its freezing/shaking sometimes.

Need your experience here to help me to solve this little issue.

https://github.com/phrizm/bs-jarallax

As i mentioned before, its pretty useful.

**Comment by crftwrk** [comment]
Hi Richard,

happy new year too. Do you have a link to your site where this image is freezing/shaking?

**Reply by ritchiemarkov** [reply]
https://www.northwestalps.com/2004/08/10/2004-canada-east-3/

**Reply by ritchiemarkov** [reply]
Please use your mobile device.

**Comment by crftwrk** [comment]
It works absolute smoothly on my iPhone12 mini? Have you tried another mobile device?

**Comment by ritchiemarkov** [comment]
Yes. I have tried and its also scattering on some points on desktop chrome
version also.



Il Sab 7 Gen 2023, 19:21 Basti ***@***.***> ha scritto:

**Reply by ritchiemarkov** [reply]
https://youtu.be/8ZJOQ-4t3Q8

**Reply by crftwrk** [reply]
I can't get that work as you described above. https://github.com/phrizm/bs-jarallax there are also missing file endings like `js` or `css` in your repo. Names are just like `jarallax.min`. Because this library not belongs to theme or any other plugin here, it should be better to open an issue on the developer site.

They have already created a WP plugin https://github.com/nk-o/jarallax/#wordpress-plugin. Why do not use this?

**Comment by ritchiemarkov** [comment, marked as answer]
Hey Basti, im trying to utilize something different and im think ukiyo.js is just perfect.
Working on it now.  FYI 
https://github.com/yitengjun/ukiyo-js
and of course chatGPT lol.

**Comment by ritchiemarkov** [comment]
https://github.com/phrizm/bs-ukijo-parallax

Please try it out

**Reply by ritchiemarkov** [reply]
Just add "parallax width-100" to the image from front
